Directory.Build.targets

```diff
@@ -6,16 +6,19 @@
     <AndroidBuildMethod>Builder.BuildAndroidIl2CPPPlayer</AndroidBuildMethod>
     <AndroidBuildPath>$(PlayerBuildPath)Android/IL2CPP_Player.apk</AndroidBuildPath>
     <!-- Assumes running `dotnet` from the root of the repo: -->
-    <UnitySampleProjectUnityVersion>$(SolutionDir)samples/unity-of-bugs/ProjectSettings/ProjectVersion.txt</UnitySampleProjectUnityVersion>
+    <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(MSBuildThisFileDirectory)'))))/</RepoRoot>
```

Comment:
So we can build individual projects (`dotnet build src/Sentry.Unity`), we couldn't rely on `SolutionDir` because that didn't expand when not buiilding the `.sln` (which it's what we're doing when we do `dotnet build` on a directory with a .sln in it).

So changing things here to be relative to the root of the repo based on this new property